### PR TITLE
Update Upgrading Wordpress Docs

### PR DESCRIPTION
**Pull request by shaneonabike**
Updated a note about the CiviCRM 4.7 civicrm.settings.php restore

### CODE REVIEW

**Comment by MikeyMJCO** [comment]
I think this is implied by the restore instruction explicitly being for versions prior to 4.6. 

If we're fixing this - drop the 4.6 steps altogether it's unsupported and it the steps are causing confusion remove the 4.6 specific steps and adjust the guide to only refer to the current way of doing it.

**Comment by MikeyMJCO** [comment]
I've committed a suggested alternative take - thoughts?

**Comment by shaneonabike** [comment]
If it's not supported let's rip it out cause it's making the docs super long and confusing. I agree that this only really applies to 4.6 cause 4.7 really doesn't need that type of backup.

**Comment by shaneonabike** [comment]
Should we not remove all 4.6 from this doc then?

**Comment by shaneonabike** [comment]
Ok I modified it again to remove all 4.6 items or make it more clear. What do you think? I'm going to create a separate issue about the localization stuff after this is done b/c I find it confusing.

**Comment by MikeyMJCO** [comment]
Not quite - we still want the upgrade from 4.6 steps - just not where it causes confusion.

File: docs/upgrade/wordpress.md [hunk 1]
```diff
@@ -106,9 +96,6 @@ Delete all files in your `templates_c` directory
 * In CiviCRM 4.7 and above:
     `<wordpress_root>/<content-dir>/uploads/civicrm/templates_c`
    
-!!! caution "For CiviCRM 4.6 and older:"
```
Comment (MikeyMJCO) [inline docs/upgrade/wordpress.md:109]: I'd keep this in.

File: docs/upgrade/wordpress.md [hunk 2]
```diff
@@ -63,15 +60,8 @@ CiviCRM will not run properly if files from previous version are present after t
...
-    If you have upgraded from a CiviCRM 4.1 version, check the contents of `civicrm.settings.php`. Make sure at the very end of the file these 2 lines exist, and if not, add them.
-    ```php
-    require_once 'CRM/Core/ClassLoader.php';
-    CRM_Core_ClassLoader::singleton()->register();
-    ```
-
-!!! note For **CiviCRM 4.6** and older:
-    For Versions installed prior to 4.7 - Restore the original `civicrm.settings.php` file from your backup into the `<wordpress_root>/wp-content/plugins/civicrm/` directory.
+!!! note "Restoring the CiviCRM settings file."
+    __Only for 4.6 and earlier__ : If you're upgrading from 4.6 or earlier you should restore the original `civicrm.settings.php` file from your backup into the `<wordpress_root>/wp-content/plugins/civicrm/` directory. For later versions, your `civicrm.settings.php` should be located in the uploads directory it is not necessary to restore this file, but you should check that it exists and has the correct settings.
```
Comment (MikeyMJCO) [inline docs/upgrade/wordpress.md]:
```suggestion
   **If you're upgrading from CiviCRM version 4.6 or earlier** you should restore the original `civicrm.settings.php` file from your backup into the `<wordpress_root>/wp-content/plugins/civicrm/` directory. **For later versions, your `civicrm.settings.php` should be located in the uploads directory** it is not necessary to restore this file, but you should check that it exists and has the correct settings.
```

File: docs/upgrade/wordpress.md [hunk 3]
```diff
@@ -27,9 +27,6 @@ Make sure you have logged in to WordPress as an administrator. Do not log out un
 
 The [before upgrading](index.md#before-upgrading) steps describe steps for backing _everything_ up in case something goes wrong during the upgrade. In addition to this important safeguard, we also need to actually _use_ the `civicrm.settings.php` settings file, as it is, during the upgrade.
 
-!!! note For **CiviCRM 4.6** and older:
```
Comment (MikeyMJCO) [inline docs/upgrade/wordpress.md:30]: And keep this.

**Comment by shaneonabike** [comment]
I reworked it and pushed back the stuff for 4.6 that was there. Sorry for the confusion. Let me know what you think.,

File: docs/upgrade/wordpress.md [hunk 4]
```diff
@@ -63,15 +63,8 @@ CiviCRM will not run properly if files from previous version are present after t
...
-    If you have upgraded from a CiviCRM 4.1 version, check the contents of `civicrm.settings.php`. Make sure at the very end of the file these 2 lines exist, and if not, add them.
-    ```php
-    require_once 'CRM/Core/ClassLoader.php';
-    CRM_Core_ClassLoader::singleton()->register();
-    ```
-
-!!! note For **CiviCRM 4.6** and older:
-    For Versions installed prior to 4.7 - Restore the original `civicrm.settings.php` file from your backup into the `<wordpress_root>/wp-content/plugins/civicrm/` directory.
+!!! note "Restoring the CiviCRM settings file."
+    __For 4.6 and earlier__ : You should restore the original `civicrm.settings.php` file from your backup into the `<wordpress_root>/wp-content/plugins/civicrm/` directory. For later versions, your `civicrm.settings.php` should be located in the uploads directory it is not necessary to restore this file, but you should check that it exists and has the correct settings.
```
Comment (MikeyMJCO) [inline docs/upgrade/wordpress.md:67]:
```suggestion
    **For 4.6 and earlier** you should restore the original `civicrm.settings.php` file from your backup into the `<wordpress_root>/wp-content/plugins/civicrm/` directory. For later versions, your `civicrm.settings.php` should be located in the uploads directory it is not necessary to restore this file, but you should check that it exists and has the correct settings.
```

File: docs/upgrade/wordpress.md [hunk 5]
```diff
@@ -103,12 +96,12 @@ If you don't have SSH access and your IP panel does not recognize tar files, you
 
 Delete all files in your `templates_c` directory
 
-* In CiviCRM 4.7 and above:
+!!! note In CiviCRM 4.7 and above:
```
Comment (MikeyMJCO) [inline docs/upgrade/wordpress.md:99]:
```suggestion
!!! note "For CiviCRM 4.7 and later"
```

File: docs/upgrade/wordpress.md [hunk 6]
```diff
@@ -103,12 +96,12 @@ If you don't have SSH access and your IP panel does not recognize tar files, you
...
     `<wordpress_root>/<content-dir>/uploads/civicrm/templates_c`
-   
-!!! caution "For CiviCRM 4.6 and older:"
-    `<wordpress_root>/<content-dir>/plugins/files/civicrm/templates_c`
 
+!!! note For CiviCRM 4.6 and older:
```
Comment (MikeyMJCO) [inline docs/upgrade/wordpress.md:102]:
```suggestion
!!! note "For CiviCRM 4.6 and older"	
```

**Comment by MikeyMJCO** [comment]
There we go! Cleared up a bit of formatting - but that's in a much better state! Going to merge!

**Comment by MikeyMJCO** [comment]
Thanks @shaneonabike :-)

**Pull request merged**